lib/urbanopt/reopt/reopt_lite_api.rb

```diff
@@ -112,7 +112,8 @@ def make_request(http, req, max_tries = 3)
             end
             tries = max_tries
           rescue StandardError => e
-            @@logger.debug("error from REopt API: #{e}")
+            @@logger.error("error from REopt API: #{e}")
+            puts "Error from REopt API: #{e}"
```

Comment:
Does this `puts` line duplicate the `logger` behavior? Using `error` level, doesn't it get printed to stdout no matter what?

Reply:
i guess so. :) you can remove 116